epicsdbbuilder/const_array.py

```diff
@@ -0,0 +1,92 @@
+from decimal import Decimal
+from .recordbase import quote_string
+from .parameter import Parameter
+
+__all__ = ['ConstArray']
+
+
+class ConstArray:
+    """Constant Link Values. EPICS Base 7.0.4.1 and above.
+
+    Example: PY Source
+    ------------------
+    `r = records.lsi('r', INP=ConstArray(['Plain String not DBLINK']))`
+
+    Example: Generated DB
+    ---------------------
+    `field(INP, ["Plain String not DBLINK"])`
+    """
+
+    def __init__(self, value):
+        self.value = value
+
+    @property
+    def value(self):
+        return self._value
+
+    @value.setter
+    def value(self, value):
+        self._assert_valid(value)
+        self._value = value
+
+    def _assert_valid(self, value, record=None, fieldname=None):
```

Comment:
What are the `record` and `fieldname` arguments for?  They're never assigned.

Reply:
The parameters were meant to be set in `Validate`. Corrected.